flask_monitoringdashboard/core/config/parser.py

```diff
@@ -23,9 +23,14 @@ def parse_version(parser, header, version):
             # The file is specified by: 'ref: <location>'
             git_file = (open(os.path.join(git, 'HEAD')).read().rsplit(': ', 1)[1]).rstrip()
             # read the git-version
-            version = open(git + '/' + git_file).read()
-            # cut version to at most 6 chars
-            return version[:6]
+            version_file = os.path.join(git , git_file)
+            if os.path.exists(version):
+                version = open(version).read()
+                # cut version to at most 6 chars
+                return version[:6]
+            else:
+                # Return "dummy" version in case of no git version file found
```

Comment:
the comment doers not make much sense anymore if we return `version` now. or? 
actually, that's a good question: what's version in this case?

Reply:
`version` is set from `APP_VERSION`. Or, in case this environment-variable, doesn't exist, it is `config.verson`.

Reply:
It use `1.0.` as pointed by 
`core/config/__init__.py:26:        self.version = '1.0'`